[2021-07-18 04:41:39,291] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_3 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 04:41:39,673] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_3 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 04:41:39,673] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 04:41:39,673] {taskinstance.py:1088} INFO - Starting attempt 3 of 3
[2021-07-18 04:41:39,674] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 04:41:39,799] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): processing_tasks.task_3> on 2021-07-17T03:39:23.918881+00:00
[2021-07-18 04:41:39,802] {standard_task_runner.py:52} INFO - Started process 1273 to run task
[2021-07-18 04:41:39,809] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'processing_tasks.task_3', '2021-07-17T03:39:23.918881+00:00', '--job-id', '32', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpfdpar6rc', '--error-file', '/tmp/tmp2pvtd1r6']
[2021-07-18 04:41:39,810] {standard_task_runner.py:77} INFO - Job 32: Subtask processing_tasks.task_3
[2021-07-18 04:41:41,023] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.processing_tasks.task_3 2021-07-17T03:39:23.918881+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-18 04:41:42,521] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T03:39:23.918881+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T03:39:23.918881+00:00
[2021-07-18 04:41:42,521] {logging_mixin.py:104} INFO - This is task 3 function
[2021-07-18 04:41:42,825] {logging_mixin.py:104} INFO - Value recived is [None]
[2021-07-18 04:41:42,825] {python.py:151} INFO - Done. Returned value was: None
[2021-07-18 04:41:43,149] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=processing_tasks.task_3, execution_date=20210717T033923, start_date=20210718T044139, end_date=20210718T044143
[2021-07-18 04:41:43,880] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-18 04:41:43,914] {local_task_job.py:151} INFO - Task exited with return code 0
